src/hospital/covid_examinations/models/worker.py

```diff
@@ -0,0 +1,18 @@
+"""Hospital worker model."""
+from django.db import models
+
+from person import Person
+from department import Department
+
+
+class Worker(Person):
```

Comment:
List the pros and cons in inheriting `Person` VS having foreignkey to `Person`

Reply:
**Inheritation** 
_Pros (Foreign Key cons)_
* When there is no intersection between Patient and Worker tables (people that are both patients and workers) - Faster searches (We don't need to search in huge Person table).
* For getting Worker name/gender/age - we should perform two queries - one, for fetch the compatible worker, second, fetch people record.
* "is a" relationship - intuitive + we will benefit Person methods without going into nested properties (weak pro).
* Inserting new worker - 2 INSERT queries - insert to People table and after insert to Worker table.

_Cons (Foreign Key Pros)_
* People duplication - in our system people can be in both tables. 
* If we change Person table we should migrate two tables instead of one.
* Update/Insert/Delete as person that is a worker and patient - 2 Queries.

I'll use Foreign Key because it prevent duplication and workers can be examined as patients also.
If you have more, please add it's interesting.

Reply:
great